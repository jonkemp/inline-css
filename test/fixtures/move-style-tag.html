<html><head><style>@media { p { color: red; } }</style></head><body><p></p></body></html>
