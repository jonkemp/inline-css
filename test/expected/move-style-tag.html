<html><head></head><body><style>
@media  {
  p {
    color: red;
  }
}
</style><p></p></body></html>
